apps/air-discount-scheme/web/screens/Admin/Admin.tsx

```diff
@@ -39,50 +88,94 @@ const Admin: Screen = ({}) => {
   }
 
   return (
-    <Layout
-      main={
-        <GridRow>
-          <GridColumn
-            span={['12/12', '12/12', '12/12', '12/12', '7/9']}
-            offset={[null, null, null, null, '1/9']}
-          >
-            <Box marginBottom={[3, 3, 3, 12]}>
-              <Panel filters={filters} />
-            </Box>
-          </GridColumn>
-        </GridRow>
-      }
-      aside={
-        <Stack space={3}>
-          <Box
-            background="purple100"
-            padding={4}
-            marginBottom={3}
-            borderRadius="standard"
-          >
-            <Box marginBottom={2}>
-              <Typography variant="h4">Síun</Typography>
-            </Box>
-            <Filters onSubmit={applyFilters} defaultValues={filters} />
-          </Box>
-          <Box
-            background="purple100"
-            padding={4}
-            marginBottom={3}
-            borderRadius="standard"
-          >
-            <Box marginBottom={2}>
-              <Typography variant="h4">Aðgerðir</Typography>
+    <GridContainer>
+      <GridRow>
+        <GridColumn
+          span={['12/12', '12/12', '7/12', '8/12', '9/12']}
+          order={[0, 0, 2, 2, 2]}
+        >
+          <Hidden above="sm">
+            {!loading && <Summary flightLegs={flightLegs} />}
+          </Hidden>
+        </GridColumn>
+        <GridColumn
+          span={['12/12', '12/12', '5/12', '4/12', '3/12']}
+          order={[1, 1, 1, 1, 1]}
+        >
+          <Stack space={3}>
+            <Box
+              background="purple100"
+              padding={4}
+              marginBottom={3}
+              borderRadius="standard"
+            >
+              <Box marginBottom={2}>
+                <Typography variant="h4">Síun</Typography>
+              </Box>
+              <Filters onSubmit={applyFilters} defaultValues={filters} />
             </Box>
-            <Box paddingTop={2}>
-              <Button width="fluid" variant="ghost">
-                Prenta yfirlit
-              </Button>
+            <Box
+              background="purple100"
+              padding={4}
+              marginBottom={3}
+              borderRadius="standard"
+            >
+              <Box marginBottom={2}>
+                <Typography variant="h4">Aðgerðir</Typography>
+              </Box>
+              <Box paddingTop={2}>
+                <Button width="fluid" variant="ghost">
+                  Prenta yfirlit
+                </Button>
+              </Box>
             </Box>
-          </Box>
-        </Stack>
-      }
-    />
+          </Stack>
+        </GridColumn>
+
+        <GridColumn
+          span={['12/12', '12/12', '7/12', '8/12', '9/12']}
+          order={[2, 2, 0, 0, 0]}
```

Comment:
I think this should be enough 
```suggestion
          order={[2, 2, 0]}
```